testsuite/forge-cli/src/main.rs

```diff
@@ -1338,17 +1344,46 @@ fn land_blocking_test_suite(duration: Duration) -> ForgeConfig<'static> {
 }
 
 // TODO: Replace land_blocking when performance reaches on par with current land_blocking
-fn land_blocking_three_region_test_suite(duration: Duration) -> ForgeConfig<'static> {
+fn land_blocking_new_test_suite(duration: Duration) -> ForgeConfig<'static> {
```

Comment:
nit: can we avoid calling things X_new?

Those names tend to stick around for a while, is there any reason not to call it land blocking?

Reply:
well, I need to make sure test is running reliably before making it land_blocking? can I do that any other way than having it running continuously for a few days?

Reply:
Yeah, renaming is fine, but possibly can we give it a permanent or descriptive name?

Eventually the new thing won't be new and it would be good to avoid renaming it again 

I accepted the PR btw these are just nits

Reply:
this is going to replace land_blocking, so that is why I called it new :)

other new tests will have appropriate names